backend/main.go

```diff
@@ -103,6 +103,7 @@ type Assignment struct {
 type Person struct {
 	ID           string  `json:"id"`
 	DisplayName  string  `json:"displayName"`
+	Location     string  `json:"location"`
```

Comment:
One thing I forgot before: can you add a test to `main_test.go` to assert that you can set and retrieve this field in the JSON as you expect?

You needn't write a whole new test case - you can just add an assertion to `TestPostPeriod`, similar to what I did for objective notes in commit https://github.com/google/peoplemath/commit/99bf1520cb64504a669e76e2736ce37d2512e6d1.